x-pack/plugin/core/template-resources/src/main/resources/entsearch/connector/elastic-connectors-mappings.json

```diff
@@ -18,6 +18,9 @@
         "api_key_id": {
           "type": "keyword"
         },
+        "connector_id": {
```

Comment:
It felt useful to add for e.g. sorting the paged response of `list` endpoint. I had issues getting `_id` to work for sorting.

Reply:
To my knowledge copying the `_id` field to a regular document field is still the [best practice](https://discuss.elastic.co/t/sort-by-id-field/169017) 👍

Reply:
I was able to sort via ID on synonyms [here](https://github.com/elastic/elasticsearch/blob/874fbbc2f05b201873edc21b5baa36753935296f/server/src/main/java/org/elasticsearch/synonyms/SynonymsManagementAPIService.java#L214).

Reply:
Great call, we've been doing this in Kibana internally for convenience and it makes everything a lot easier. Doing this in the API makes a lot of sense.